src/inputs/Text.vue

```diff
@@ -1,14 +1,43 @@
 <template>
-    <div class="input">
-        Input
+    <div class="text">
+        <label>{{ this.label }}</label>
+        <input v-model="superText"
+        :type="this.type" 
+        :id="this.id" 
+        :placeholder="this.placeholder"
+        :name="this.name" 
+        :value="this.value"/> 
     </div>
 </template>
 
 <script>
     import Vue from 'vue';
+    import _ from 'lodash';
+    import inputProps from '../inputProps';
 
     export default {
         name: 'SuperText',
+        data() {
+            return {
+                superText,
+            };
+        },
+        props: _.assign({
+            id: {
```

Comment:
Why do we need this prop?

Reply:
When people want to use this libary and uses id's for whatever reason then the option is open.